skills/gitlab-mcp/reference/pipelines.md

```
@@ -45,6 +45,18 @@ list_deployments           -> list deployments with filters
 list_environments          -> list project environments
 get_environment            -> environment details
+update_environment         -> update environment settings
+delete_environment         -> delete a stopped environment
+stop_environment           -> stop one environment
+stop_stale_environments    -> stop environments older than a cutoff
+delete_review_app_environments -> clean up stopped review apps
+```
+
+## Pipeline Triggers
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)